src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -1,9 +1,9 @@
 import classNames from "classnames";
 import React, { useLayoutEffect, useRef } from "react";
+import { Row } from "react-table";
```

Comment:
I'm not sure why vscode moved this import but I think it make sense since is grouping the library imports at the top